src/liberty/dashboard.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseLibertyProject } from "./baseLibertyProject";
+
+/**
+ * Copyright (c) 2022 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
```

Comment:
Copyright should be above the import statement

Reply:
fixed